src/components/HeadTag.js

```diff
@@ -0,0 +1,10 @@
+import Head from 'next/head';
+
+export default function HeadTag({ title }) {
+  return (
+    <Head>
+      <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+      <title>{title}</title>
```

Comment:
Can we add a default title here `Treetracker by Greenstand` and if we provide a `title` we make it `{title} | Treetracker by Greenstand`. This will reduce duplicate code